src/PanelTraits/Boxes.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Backpack\CRUD\PanelTraits;
+
+trait Boxes
+{
+    public $sideBoxesEnabled = false;
+    public $boxesOptions = [];
+
+    public function enableSideBoxes()
```

Comment:
OK, but this is the same methods as Tabs trait : https://github.com/Laravel-Backpack/CRUD/blob/master/src/PanelTraits/Tabs.php#L10